tests/_multi_screen.lua

```diff
@@ -411,6 +411,7 @@ local function add_steps(real_steps, new_steps)
     for _, dispo in ipairs(dispositions) do
         -- Cleanup
         table.insert(real_steps, function()
+            collectgarbage("collect")
             if #client.get() == 0 then return true end
```

Comment:
Pulled out here, and changed accordingly: https://github.com/awesomeWM/awesome/pull/1333.